lib/api/formtools/grid.js

```diff
@@ -0,0 +1,263 @@
+var linz = require('../../../');
+    async = require('async'),
+    clone = require('clone');
+
+(function  () {
+
+    function getSettings(modelName, cb) {
```

Comment:
As per our discussion, it would be good to get rid of this and start making use of the model bootstrapping we have in place which places the grid settings in `model.linz.formtools.grid`.